design/KruizeLocalAPI.md

```diff
@@ -2155,6 +2155,88 @@ see [Create Experiment](/design/CreateExperiment.md)
 
 </details>
 
+**Request with `experiment_type` field**
```

Comment:
@shekhar316 - Recommendations output also needs to be updated

Reply:
Added the response for generateRecommendations API.